react/src/lib/Button/index.js

```diff
@@ -266,6 +270,7 @@ Button.defaultProps = {
   active: false,
   ariaLabel: '',
   ariaLabelledBy: '',
+  ariaPressed: undefined,
```

Comment:
Should this really be undefined?

Reply:
For a non-toggle button it shouldn't have ariaPressed, not even false

Reply:
I was referring to an empty string vs undefined

Reply:
Okay I can change that real quick